### PR TITLE
Edits to BigAnimal PR5422

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx [hunk 1]
```diff
@@ -97,11 +99,10 @@ To delete a write lead node from the cluster:
 Don't delete the Azure Kubernetes VMSS here or sub resources directly.
 !!!
 
-7. Browse to the Data Plane, choose Workloads, and locate the Kubernetes resources for your cluster to delete a chosen node.
+7. To delete a chosen node, browse to the data plane, select **Workloads**, and locate the Kubernetes resources for your cluster.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx:102]: I'm not sure if this edit is correct. I didn't see an action that would actually perform the delete. The edit is basically "goal before action,," but I'm not 100% sure that's what you meant.

File: product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx [hunk 2]
```diff
@@ -68,27 +69,28 @@ p-x67kjp3fsq-a  2456382099  data     world   p-x67kjp3fsq-a-1
...
-4. Choose the cluster to test fault injection with and copy the string value from the URL. The string value is located after the underscore.  
+3. In the left navigation of BigAnimal portal, select **Clusters**. 
+4. Select the cluster to test fault injection with and copy the string value from the URL. The string value is located after the underscore.  
 
      ![Delete a write lead](images/biganimal_faultinjectiontest_1.png)
 
 
-5. In your Azure subscription, paste the string into the search and prefix it with **dp-** to search for the data plane.  
-   * From the results, choose the Kubernetes service from the Azure Region that your cluster is deployed in.
+5. To search for the data plane, in your Azure subscription, paste the string into the search and prefix it with `dp-`.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx:86]: This really needed to be its own step. It's not really a bullet item of the pervious step. You could have made it all one step.

Also, I recommend just using the number 1. for steps so you don't have to reorder if you decide to add steps. The steps come out in order if they are all numbered 1.

**Comment by ebgitelman** [comment]
@piano35-edb This overall looked very good! I think most of these edits are self-evident what they were for...Several are "goal before action." When you're reading along and it says, Do such and such to do this or that, when you get to the "this or that" it's often jarring to find out why you were doing what you're doing at the end of the sentence instead of the beginning. I don't 100% always make this edit, but generally in steps I would.

I had one query, as I wasn't 100% sure my edit was correct or if the procedure might be missing something.

File: product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx (hunk 1 above)
Comment (piano35-edb) [reply product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx:102]: Thank you.  I have a draft copy of this to collaborate with the SDE on additional changes.  I'll add your edits into my draft before publishing.

Comment (ebgitelman) [reply product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx:102]: Okay, let me know when it's done and I'll cancel this pull request, assuming you meant to transfer all the edits.

**Review by piano35-edb — APPROVED** [review]

**Pull request merged**